plugins/postcss-conditional-values/README.md

```diff
@@ -0,0 +1,149 @@
+# PostCSS Conditional Values [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS Logo" width="90" height="90" align="right">][postcss]
+
+[<img alt="npm version" src="https://img.shields.io/npm/v/@csstools/postcss-conditional-values.svg" height="20">][npm-url] [<img alt="Build Status" src="https://github.com/csstools/postcss-plugins/workflows/test/badge.svg" height="20">][cli-url] [<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+[PostCSS Conditional Values] lets you easily apply space toggle hacks with some syntactic sugar.
+
+This plugin adds a non-standard function : `csstools-if( else )` which acts as a ternary operator.
+
+`csstools-if(--a-variable <value-when-true> else <value-when-false>)`
+
+You control the outcome by setting `--a-variable` to `true` and `false`.
+
+```pcss
+.fancy-container {
+	--is-fancy: true;
+}
+
+.block {
+	color: csstools-if(--is-fancy pink else red);
+}
+
+/* becomes */
+
+:root {
+	--is-fancy:  ;
+}
+
+.fancy-container {
+	--is-fancy: initial;
+}
+
+.block {
+	--is-fancy--0: var(--is-fancy) red;
+	color: var(--is-fancy--0,pink);
+}
+```
+
+For more information see one of the articles :
```

Comment:
Feel free to change the suggestion felt odd that it was making you choose.

```suggestion
For more information on how the trick works, you can read more on these articles:
```

Reply:
Good catch!